src/app/meet/[meetingId]/page.tsx

```
@@ -0,0 +1,633 @@
+					className="absolute top-0 left-0 right-0 h-14 z-40 flex items-center justify-between px-6"
+					style={{
+						background:
+							"linear-gradient(to bottom, rgba(10,10,18,0.8), transparent)",
+					}}
+				>
+					<div className="flex items-center gap-3">
+						<div className="flex -space-x-2">
+							{[user].map((u, i) => (
+								<div
+									className="w-8 h-8 rounded-full border-2 border-[#0a0a12] bg-indigo-500 flex items-center justify-center text-[10px] font-bold shadow-lg"
+									key={i}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)